README.md

```diff
@@ -312,6 +312,7 @@ These rules are only relevant to ES6 environments.
 |:ballot_box_with_check:|[object-shorthand](http://eslint.org/docs/rules/object-shorthand)|[object-literal-shorthand](https://palantir.github.io/tslint/rules/object-literal-shorthand)|require method and property shorthand syntax for object literals|
 |:white_check_mark:|[prefer-arrow-callback](http://eslint.org/docs/rules/prefer-arrow-callback)|[ter-prefer-arrow-callback](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terPreferArrowCallbackRule.md)|require arrow functions as callbacks|
 |:ballot_box_with_check:|[prefer-const](http://eslint.org/docs/rules/prefer-const)|[prefer-const](https://palantir.github.io/tslint/rules/prefer-const)|suggest using `const` declaration for variables that are never modified after declared|
+|:x:|[prefer-destructuring](http://eslint.org/docs/rules/prefer-destructuring)|prefer-destructuring|prefer destructuring from arrays and objects|
```

Comment:
I’m getting an error rebuilding the README:

```
[16:28:39] Using gulpfile ~/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/gulpfile.js
[16:28:39] Starting 'lint'...
Warning: The 'no-use-before-declare' rule requires type checking
[16:28:41] Finished 'lint' after 1.71 s
[16:28:41] Starting 'build'...
/Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/node_modules/tsutils/src/typeguard.d.ts(140,70): error TS2694: Namespace 'ts' has no exported member 'EnumLiteralType'.
src/readme/index.ts(51,40): error TS2559: Type '"utf8"' has no properties in common with type '{ encoding?: string | undefined; mode?: string | undefined; flag?: string | undefined; }'.
src/readme/index.ts(51,49): error TS7006: Parameter 'writeErr' implicitly has an 'any' type.
src/readme/index.ts(126,42): error TS2559: Type '"utf8"' has no properties in common with type '{ encoding?: string | undefined; mode?: string | undefined; flag?: string | undefined; }'.
src/readme/index.ts(126,51): error TS7006: Parameter 'writeErr' implicitly has an 'any' type.
[16:28:44] TypeScript: 5 semantic errors
[16:28:44] TypeScript: emit succeeded (with errors)
[16:28:44] 'build' errored after 2.99 s
[16:28:44] Error: TypeScript has reported errors
    at formatError (/Users/hayfields/.nvm/versions/node/v7.9.0/lib/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/Users/hayfields/.nvm/versions/node/v7.9.0/lib/node_modules/gulp/bin/gulp.js:195:15)
    at emitOne (events.js:96:13)
    at Gulp.emit (events.js:191:7)
    at Gulp.Orchestrator._emitTaskDone (/Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/node_modules/orchestrator/index.js:264:8)
    at /Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/node_modules/orchestrator/index.js:275:23
    at finish (/Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/node_modules/orchestrator/lib/runTask.js:29:3)
    at DestroyableTransform.<anonymous> (/Users/hayfields/Documents/Jed_Folder/github-clones/Forks/tslint-eslint-rules/gulpfile.js:90:9)
    at emitNone (events.js:91:20)
```

It looks like `@types/node` expects the parameter [here](https://github.com/buzinas/tslint-eslint-rules/blob/166476060e1b085b3e00bd445926ff13e59a992f/src/readme/index.ts#L51) to be an options object, but the node [docs](http://devdocs.io/node~6_lts/fs#fs_fs_writefile_file_data_options_callback) state that it can be a string that specifies the encoding.